src/components/Button/Button.stories.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { storiesOf } from '@storybook/react';
+import { decorateAction } from '@storybook/addon-actions';
+// import { action } from '@storybook/addon-actions';
+import { withInfo } from '@storybook/addon-info';
+import Button from './Button';
+
+const targetAction = decorateAction([
+  args => {
+    console.log(args[0].target);
+    return [args[0].target];
+  },
+]);
+console.log('decorate', decorateAction, targetAction);
+
+const SIZES = ['small', 'medium', 'large'];
+
+storiesOf('Button', module)
+  .add(
+    'default',
+    withInfo(`
+    Basic usage of Button Component
+    `)(() => <Button onClick={targetAction('clicked')}>Hello Button</Button>)

```

Comment:
I wonder if we need a `withInfo` for _every_ story... I think in many cases it's very helpful, but I'm wary of having too much friction when creating stories.

For example, I think for the stories in this file, only "type" needs additional info, since it's not immediately clear what that means.